onnxruntime/server/grpc/grpc_app.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <grpcpp/grpcpp.h>
+#include "prediction_service_impl.h"
+#include "../environment.h"
```

Comment:
I'm not a fan of using `..` in includes. Is the problem that environment.h is not part of a library so you can't use target_include_directories?